From 3f1b0d39c3a1070f14d6f9f0d18ea79db20d5eb9 Mon Sep 17 00:00:00 2001
From: Andreas Pokorny <andreas.pokorny@canonical.com>
Date: Fri, 18 Nov 2016 13:48:04 +0100
Subject: [PATCH libinput 4/4] Keep stanza for new symbols on version 0.22.0

This is necessary to stay compatible with previous mir server
---
 src/libinput.sym | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/src/libinput.sym b/src/libinput.sym
index b4b3599..2bbbe2c 100644
--- a/src/libinput.sym
+++ b/src/libinput.sym
@@ -174,12 +174,25 @@ LIBINPUT_0.21.0 {
 	libinput_event_touch_get_time_usec;
 } LIBINPUT_0.20.0;
 
+LIBINPUT_0.22.0 {
+       libinput_event_touch_get_major;
+       libinput_event_touch_get_major_transformed;
+       libinput_event_touch_get_minor;
+       libinput_event_touch_get_minor_transformed;
+       libinput_event_touch_get_orientation;
+       libinput_event_touch_get_pressure;
+       libinput_event_touch_has_major;
+       libinput_event_touch_has_minor;
+       libinput_event_touch_has_orientation;
+       libinput_event_touch_has_pressure;
+} LIBINPUT_0.21.0;
+
 LIBINPUT_1.1 {
 	libinput_device_config_accel_get_profile;
 	libinput_device_config_accel_get_profiles;
 	libinput_device_config_accel_get_default_profile;
 	libinput_device_config_accel_set_profile;
-} LIBINPUT_0.21.0;
+} LIBINPUT_0.22.0;
 
 LIBINPUT_1.2 {
 	libinput_device_config_tap_get_drag_enabled;
@@ -280,16 +293,3 @@ LIBINPUT_1.5 {
 	libinput_device_config_tap_get_default_button_map;
 	libinput_device_config_tap_set_button_map;
 } LIBINPUT_1.4;
-
-LIBINPUT_1.6_unreleased {
-	libinput_event_touch_get_major;
-	libinput_event_touch_get_major_transformed;
-	libinput_event_touch_get_minor;
-	libinput_event_touch_get_minor_transformed;
-	libinput_event_touch_get_orientation;
-	libinput_event_touch_get_pressure;
-	libinput_event_touch_has_major;
-	libinput_event_touch_has_minor;
-	libinput_event_touch_has_orientation;
-	libinput_event_touch_has_pressure;
-} LIBINPUT_1.5;
-- 
2.9.3

